### ISSUE TITLE
Get predictions from a model

### ISSUE THREAD

**Issue by china-zyy** [heartexlabs/label-studio#1443]
how to predict all tasks? not only predict the current page

**Comment by smoreface** [comment]
@china-zyy did you check https://labelstud.io/guide/ml.html#Get-predictions-from-a-model ?

**Comment by makseq** [comment]
```
 For a large dataset, the HTTP request to retrieve predictions might be interrupted by a timeout. If you want to get all predictions for all tasks in a dataset from connected machine learning backends, make a POST call to the predictions endpoint of the Label Studio API to prompt the machine learning backend to create predictions for the tasks.
```

**Comment by china-zyy** [comment]
> From docs:
> `For a large dataset, the HTTP request to retrieve predictions might be interrupted by a timeout. If you want to get all predictions for all tasks in a dataset from connected machine learning backends, make a POST call to the predictions endpoint of the Label Studio API to prompt the machine learning backend to create predictions for the tasks.`

yeah, got it, but the document api.md in docs/source/guide have a sentence "These API endpoints were introduced in Label Studio version 0.8.1 and are only valid until version 0.9.1." so, the version 1.1 don't support these api ?
![image](https://user-images.githubusercontent.com/50455382/133361503-95b57dbf-0c9c-44fd-af45-fd6033e18531.png)

**Comment by china-zyy** [comment]
> From docs:
> `For a large dataset, the HTTP request to retrieve predictions might be interrupted by a timeout. If you want to get all predictions for all tasks in a dataset from connected machine learning backends, make a POST call to the predictions endpoint of the Label Studio API to prompt the machine learning backend to create predictions for the tasks.`
I change the code "evaluate_predictions(tasks_for_predictions)" to  "evaluate_predictions(queryset.filter(predictions__isnull=True))", then I got all tasks prediction in the web UI, but I thick It's terrible.
![image](https://user-images.githubusercontent.com/50455382/133361801-6336e4cc-0946-4196-8743-8b8622e66a05.png)

**Comment by smoreface** [comment]
@china-zyy 
API docs are here, linked from the API page that you're referencing:
https://labelstud.io/api#operation/api_predictions_read

**Comment by china-zyy** [comment]
> @china-zyy
> API docs are here, linked from the API page that you're referencing:
> https://labelstud.io/api#operation/api_predictions_read
 
thank you, I sent post request to http://localhost:9090/predict , body is {"tasks": [{"data": {"data": "text"}}], "model_version": "1630929480", "project": "39.1630897128", "label_config": "<View>\n  <Text name='text' value='$data'/>\n  <View style='box-shadow: 2px 2px 5px #999; padding: 20px; margin-top: 2em; border-radius: 5px;'>\n    <Header value='Choose text classification'/>\n    <Choices name='sentiment' toName='text' choice='single' showInLine='true'>\n      <Choice value='非遗'/>\n      <Choice value='其他'/>\n    </Choices>\n  </View>\n</View>"}, then I can get the text prediction, I want to know, how I can get the all tasks prediction in the label-studio http://localhost:8080/....
